examples/recurrence_utils.py

```diff
@@ -1,13 +1,26 @@
+import calendar
 import sys
 from collections import namedtuple
 from datetime import date, datetime, timedelta
 from enum import IntEnum
 from typing import List, Optional, Set, Tuple, Union
 
+from django.utils.translation import gettext_lazy as _
 from versio.version import Version
 from versio.version_scheme import Pep440VersionScheme
 
 
+def locale_weekdays():
+    return ["ho"] + [_(day).lower()[:2] for day in calendar.day_abbr]
+
+
+date_adjectives = ("1st", "first", "2nd", "second", "3rd", "third", "4th", "fourth", "5th", "fifth", "last")
+
+
+def locale_adjectives():
+    return [_(adj)[:3] for adj in date_adjectives]
```

Comment:
A si tole probal narest makemessages? To naredi vse stringe? Ker po mojem jih ne...